Contents/Sketch/typography.js

```diff
@@ -19,8 +22,11 @@ var applyTypography = function (newStyles, sharedStyles) {
     var textLayer = [[MSTextLayer alloc] initWithFrame:nil];
 
     if("Size"      in style)  { textLayer.setFontSize(style.Size); }
-    if("Line"      in style)  { textLayer.setLineSpacing(style.Line); }
-    if("Character" in style)  { textLayer.setCharacterSpacing(style.Character); }
+    if("Line"      in style)  { textLayer.setLineHeight(style.Line); }
+    if("Character" in style)  { 
+      var characterSpacing = Number(style.Character);
+      textLayer.setCharacterSpacing(characterSpacing); 
+    }
```

Comment:
Cool, this is better than my fix (which I had been begging people to test before merging in) https://github.com/nolastan/sync.sketchplugin/compare/character-fix